component/component/vshn_postgres.jsonnet

```diff
@@ -630,7 +628,6 @@ local sgObjectStorage = {
     comp.FromCompositeFieldPathWithTransformPrefix('metadata.labels[crossplane.io/composite]', 'spec.forProvider.manifest.metadata.name', 'sgbackup'),
     comp.FromCompositeFieldPathWithTransformPrefix('metadata.labels[crossplane.io/composite]', 'spec.forProvider.manifest.metadata.namespace', 'vshn-postgresql'),
     comp.FromCompositeFieldPath('metadata.labels[crossplane.io/composite]', 'spec.forProvider.manifest.spec.s3Compatible.bucket'),
-    comp.FromCompositeFieldPath('metadata.labels[crossplane.io/claim-namespace]', 'spec.forProvider.spec.writeConnectionSecretToRef.namespace'),
```

Comment:
Why has this patch been removed?

Reply:
Because this patch was always broken, it tries to patch something that doesn't exist. Previous Crossplane versions just discarded a broken patch, the new function based implementation is stricter.

```
cannot compose resources: cannot apply composed resource "sg-backup": failed to create typed patch object (/pgsql-app1-prod-ggtd4-object-storage; kubernetes.crossplane.io/v1alpha1, Kind=Object): .spec.forProvider.spec: field not declared in schema
```